internal/config/config_type.go

```diff
@@ -42,13 +42,45 @@ var configOptions = []ConfigOption{
 	},
 }
 
-func ConfigOptions() []ConfigOption {
+func ConfigOptions() []configOption {
```

Comment:
This is an exported method that returns an un-exported struct. This might make it more difficult for the caller to handle the result

Reply:
Good point. I will fix that.